spec/lib/inventory_item_response_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'spec_helper'
+require 'gman_client'
+require 'vcr'
+require 'httparty'
+require 'webmock'
+require 'support/vcr'
+
+RSpec.describe GmanClient do
+  describe '.inventory_items' do
+    let(:response) do
+      VCR.use_cassette('inventory_items') do
+        GmanClient.inventory_items
+      end
+    end
+
+    subject(:client_response) { response }
+    let(:driver_commission) { client_response.first }
+
+    it 'client response will not be empty' do
+      expect(client_response.empty?).to be_falsey
```

Comment:
I guess I could do: 
```
    it 'client response will be truthy' do
      is_expected.to be_falsey
    end
```
But that will bring us back to our original problem of the test always passing. what do you have in mind?